meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/imx-atf/0001-plat-imx8m-obtain-boot-set-from-bootrom-even-log.patch

```diff
@@ -0,0 +1,96 @@
+From 58cd2feeb07e496c7884f6655f258eca2a25b7ff Mon Sep 17 00:00:00 2001
+From: Igor Opaniuk <igor.opaniuk@foundries.io>
+Date: Fri, 18 Mar 2022 14:52:22 -0300
+Subject: [PATCH] plat: imx8m: obtain boot set from bootrom even log
+
+Both i.MX8MP and i.MX8MN don't have a dedicated bit which indicates
+a boot image set booted by BootROMv According to AN12853 [1] i.MX ROMs Log
+Events, it is possible to determine whether fallback event occurred
+by parsing the ROM event log. In case ROM event ID 0x51 is present,
+fallback event did occur.
+
+The code is based on U-Boot implementation [2].
+
+[1]: https://www.nxp.com/webapp/Download?colCode=AN12853
+[2]: https://github.com/u-boot/u-boot/commit/a5ee05cf7180b411ffdf148ca8cb220c029f2e19
+Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>
```

Comment:
Can you please add the upstream-status tag?

Reply:
I'll send it upstream tomorrow, and will add a tag

Reply:
added